src/routes/(admin)/questionbank/single-question/[id]/+page.svelte

```
@@ -0,0 +1,200 @@
+			$formData.correct_answer = totalOptions
+				.filter((option) => option.correct_answer)
+				.map((option) => option.id);
+			$formData.created_by_id = data.user.id;
+			$formData.organization_id = data.user.organization_id;
+		}
+	});
+
+	questionData &&
+		($formData.state_ids = questionData.locations?.map((state: any) => {
+			return String(state.state_id);
+		}));
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)